crawl-ref/source/ghost.cc

```diff
@@ -95,7 +96,6 @@ static spell_type search_order_misc[] =
     SPELL_PETRIFY,
     SPELL_POLYMORPH,
     SPELL_FORCE_LANCE,
-    SPELL_SLOW,
```

Comment:
Don't include a sneaky change like this. Slow checks MR and should stay in the misc spells.

Reply:
It wasn't to be sneaky! I put it into its own commit. I'll drop from this PR.